tests/env_file_test.conf

```diff
@@ -1,3 +1,4 @@
+# Ignored comment
 VAR_OTHER=file
-
+# Comment
```

Comment:
If we're gonna support `;` style comments, we might as well throw in a test for it as well.